campaigns.py

```diff
@@ -6,52 +6,86 @@
+        'extra_info': prompt("Extra information for campaign:", key='extra_info'),
+        'more_info': prompt("Link for more information:", key='more_info'),
+        'campaign_class': prompt("Campaign class:", key='campaign_class', validate=validate_classes)
     }
 
-    template = Template("""<div id="campaign" class="{{ campaign_class }}">
-  <div class="campaign-inner">
-    <h1>{{ heading|e }}</h1>
-    <p>{{ extra_info|e }}</p>
-    <a href="{{ more_info|e }}">More information</a>
-  </div>
-</div>""")
-    env['template_contents'] = template.render(context)
 
-    print "Template contents:\n%s" % env['template_contents']
+def template(app):
+    if app == 'frontend':
+      template = Template("""<div id="campaign" class="{{ campaign_class }}">
+    <div class="campaign-inner">
+      <h1>{{ heading|e }}</h1>
+      <p>{{ extra_info|e }}</p>
+      <a href="{{ more_info|e }}">More information</a>
+    </div>
+  </div>""")
+    elif app == 'static':
+      template = Template("""<p>{{ heading|e }}<br />
+    {{ extra_info|e }}</p>
+  <a href="#" class="right">{{ more_info|e }}</a>""")
+
+    env['template_contents'] = template.render(env.context)
+
 
 @task
 @roles('class-frontend')
-def deploy_to_homepage():
-    execute(homepage_template)
+def banner(application, action):
+    if action == 'deploy':
+        execute(template, application)
+        if application == 'frontend':
+            remote_filenames = ['/var/apps/frontend/app/views/root/_campaign_notification.html.erb']
+        elif application == 'static':
+            remote_filenames = ["/var/apps/static/app/views/notifications/banner_{}.erb".format(env.campaign_class)]
+        content = env['template_contents']
+    elif action == 'remove':
+        if application == 'frontend':
+            remote_filenames = ['/var/apps/frontend/app/views/root/_campaign_notification.html.erb']
+        elif application == 'static':
+            remote_filenames = ["/var/apps/static/app/views/notifications/banner_%s.erb" % i for i in CAMPAIGN_CLASSES]
+        content = ''
+    for remote_filename in remote_filenames:
+        put(StringIO.StringIO(content), remote_filename, use_sudo=True, mirror_local_mode=True)
+        sudo('chown deploy:deploy %s' % remote_filename)
+    execute(app.reload, application)
 
-    contents = env['template_contents']
 
-    remote_filename = '/var/apps/frontend/app/views/root/_campaign_notification.html.erb'
-    put(StringIO.StringIO(contents), remote_filename, use_sudo=True, mirror_local_mode=True)
-    sudo('chown deploy:deploy %s' % remote_filename)
-    execute(vm.reload_unicorn, name='frontend')
+@task
+@roles('class-frontend')
+def deploy_emergency_banner():
+    """Deploy an emergency banner to GOV.UK"""
+    execute(set_context)
+    for application in APPLICATIONS:
+        if application == 'static' and env.campaign_class == 'black':
+            return 'A campaign_class of {} only gets put on frontend..skipping..'.format(env.campaign_class)
```

Comment:
I don't think we need to output a message to the user at this point as they don't need to know the implementation details when running the task.